bin/main

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+
+winner = false
+draw = false
+
+spots = %w[1 2 3 4 5 6 7 8 9]
+puts 'Enter name for the first player'
+player1 = 'player1'
+puts 'Enter name for the second player'
+player2 = 'player2'
+
+board = "\n[1] [2] [3]\n[4] [5] [6]\n[7] [8] [9]\n"
+
+puts board
+
+validator = false
+
+while winner == false || draw == false
```

Comment:
Something like this will be better
```rb
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```